pkg/network/protocols/http2/protocol.go

```diff
@@ -211,13 +213,23 @@ func (p *protocol) PreStart(mgr *manager.Manager) (err error) {
 func (p *protocol) PostStart(mgr *manager.Manager) error {
 	// Setup map cleaner after manager start.
 	ticker := time.NewTicker(10 * time.Second)
+	http2Telemetry := &HTTP2Telemetry{}
+	p.done = make(chan struct{})
+	var zero uint32
+
+	mp, _, err := mgr.GetMap(telemetryMap)
+	if err != nil {
+		log.Warnf("error retrieving http2 telemetry map: %s", err)
```

Comment:
you should abort here as `mp` is nil

Reply:
I wanted to make sure that the setupHTTP2InFlightMapCleaner and setupDynamicTableMapCleaner would still happen. so I have changed to code and as you said retuned if I do not have the map